seisbench/data/geofon.py

```diff
@@ -23,8 +24,65 @@ class GEOFON(BenchmarkDataset):
     GEOFON dataset consisting of both regional and teleseismic picks. Mostly contains P arrivals,
     but a few S arrivals are annotated as well. Contains data from 2010-2013. The dataset will be
     downloaded from the SeisBench repository on first usage.
+
```

Comment:
Thanks for adding the extensive docstring. I think it would be good to put a subheading in between to make clear these details refer to the source data for converting the dataset from scratch.